### PR TITLE
ENH background the solver to save memory

### LINKED ISSUE #311: update me endpoint spikes memory usage
It appears the update me endpoint is spiking memory usage.

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Pushed the branch to main repo
* [ ] CI passed on the branch

Closes #311

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,8 +9,7 @@ jobs:
       - run:
           name: update web services
           command: |
-            ret=`curl -H "CF_WEBSERVICES_TOKEN: ${CF_WEBSERVICES_TOKEN}" -X POST https://conda-forge.herokuapp.com/conda-webservice-update/cron`
-            echo $ret | grep "received"
+            curl -H "CF_WEBSERVICES_TOKEN: ${CF_WEBSERVICES_TOKEN}" -X POST https://conda-forge.herokuapp.com/conda-webservice-update/cron
```
Comment (beckermr) [inline .circleci/config.yml]: This check basically always fails due to incoming requests being in a queue. So I got rid of it.

File: conda_forge_webservices/update_me.py [hunk 2]
```diff
@@ -36,8 +44,6 @@ def update_me():
             os.environ['GH_TOKEN'], repo_name)
 
         repo = Repo.clone_from(url, clone_dir)
-        msg_vers = ", ".join(["{}={}".format(k, v) for k,v in to_install.items()])
-        author = Actor("conda-forge-admin", "pelson.pub+conda-forge@gmail.com")
```
Comment (beckermr) [inline conda_forge_webservices/update_me.py]: the global git config defines the actor so this is not needed.

**Comment by beckermr** [comment]
Alright, this works locally. Going to merge!

**Comment by beckermr** [comment]
Ahhhhh. I think this PR works, but I hit an outage: https://status.heroku.com/incidents/1954

I will rerevert my revert when the outage is solved

**Comment by jakirkham** [comment]
Sounds like things started working ~6mins ago. So should be good to go 🙂

**Pull request merged**